src/DynamoCoreWpf/ViewModels/Search/SearchCategory.cs

```diff
@@ -126,7 +130,7 @@ public bool Visibility
 
         public bool IsSelected
         {
-            get { return true; }
+            get { return false; }
         }
```

Comment:
These `IsSelected` prevent to select top result. Treeview can't have multiple selection. First we select top result item, but them, when we add new category or group, treeview selects added item and we lose top result selection. When top result wasn't part of the whole collection, everything worked well.